docs/INSTALLATION.md

```diff
@@ -32,4 +32,3 @@ that will give you the Visual Studio 2013 IDE and the latest version of the
 Microsoft Visual C compiler.
 
 [vs-1013]: http://www.visualstudio.com/en-us/products/visual-studio-community-vs
```

Comment:
Can we change this to not reference a specific version of Visual Studio? Just point the link to visualstudio.com (since I think the link keeps changing) and say that Microsoft offers a free version of the Visual Studio IDE that comes with everything you need (compiler/etc).

Actually I have no idea how easy it is to use VS for these Makefile projects, do you know?